src/test/java/com/google/cloud/flink/bigquery/ArrowSchemaConverterTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.flink.bigquery;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.flink.bigquery.arrow.util.ArrowSchemaConverter;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.Schema;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.table.types.logical.RowType.RowField;
+import org.junit.Test;
+
+public class ArrowSchemaConverterTest {
+  @Test
+  public void convertToSchemafromRowTypeTest() {
+
+    List<RowField> fields = new ArrayList<RowField>();
+    fields.add(new RowField("int_dataType", DataTypes.BIGINT().getLogicalType()));
+    fields.add(new RowField("string_dataType", DataTypes.STRING().getLogicalType()));
+    fields.add(
+        new RowField("array_dataType", DataTypes.ARRAY(DataTypes.STRING()).getLogicalType()));
+    fields.add(new RowField("boolean_dataType", DataTypes.BOOLEAN().getLogicalType()));
+    fields.add(new RowField("bytes_dataType", DataTypes.BYTES().getLogicalType()));
+    fields.add(new RowField("date_dataType", DataTypes.DATE().getLogicalType()));
+    fields.add(new RowField("decimal_dataType", DataTypes.DECIMAL(10, 0).getLogicalType()));
+    fields.add(new RowField("double_dataType", DataTypes.DOUBLE().getLogicalType()));
+    fields.add(new RowField("row_dataType", DataTypes.ROW().getLogicalType()));
+    fields.add(new RowField("timestamp_dataType", DataTypes.TIMESTAMP().getLogicalType()));
+    fields.add(new RowField("time_dataType", DataTypes.TIME().getLogicalType()));
+    RowType rowType = new RowType(fields);
+    Schema schema = ArrowSchemaConverter.convertToSchema(rowType);
+    assertThat(schema).isNotNull();
+    assertThat(schema.toString())
```

Comment:
Please test this by querying the fields rather than the toString()

Reply:
Addressed